plugins/vendor/zoom/infra/kZoomClient.php

```diff
@@ -150,17 +150,23 @@ public function retrieveMeeting($meetingId)
 	public function retrieveTrackingField($meetingId)
 	{
 		$meeting = $this->retrieveMeeting($meetingId);
+		$categoryName = '';
+		$categoryPath = '';
 		if ($meeting && isset($meeting[kZoomRecording::TRACKING_FIELDS]))
 		{
 			foreach ($meeting[kZoomRecording::TRACKING_FIELDS] as $trackingField)
 			{
 				if ($trackingField[kZoomRecording::FIELD] === kZoomRecording::KALTURA_CATEGORY)
 				{
-					return $trackingField[kZoomRecording::VALUE];
+					$categoryName = $trackingField[kZoomRecording::VALUE];
+				}
+				if ($trackingField[kZoomRecording::FIELD] === kZoomRecording::KALTURA_CATEGORY_PATH)
+				{
+					$categoryPath = $trackingField[kZoomRecording::VALUE];
 				}
 			}
 		}
-		return null;
+		return $categoryPath . $categoryName;
```

Comment:
1. if $categoryPath does not ends with > , lets add it.
2. Add logs so it will be easy to handle customer complains